CMakeLists.txt

```diff
@@ -6,8 +6,13 @@ enable_testing()
 
 option(TRACCC_UNIT_TESTS "Enable unit tests" On)
 option(TRACCC_BENCHMARKS "Enable benchmark tests" On)
+option(TRACCC_BUILD_CUDA "Build CUDA" ON)
```

Comment:
This should probably be OFF by default. Not everyone who is interested in traccc has a working CUDA setup, and builds that fail / require exotic deps by default are not nice.